Nighttime-Operations/Observatory/DIMM/index.rst

```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope. 
```

Comment:
Maybe add that the wedge results in a displacement of one image on the sensor?

Reply:
Have modified syntax. Let me know if it makes sense.